app/v3/asset/internal/keeper/keeper.go

```diff
@@ -38,7 +38,7 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 	return ctx.Logger().With("module", fmt.Sprintf("%s/%s", "iris", types.MsgRoute))
 }
 
-// IssueToken issue a new token
+// IssueToken issues a new token
 func (k Keeper) IssueToken(ctx sdk.Context, msg types.MsgIssueToken) (sdk.Tags, sdk.Error) {
 	scale := int(msg.Decimal)
 	token := types.NewFungibleToken(msg.Symbol, msg.Name, msg.MinUnitAlias, msg.Decimal,
```

Comment:
When calling `NewFungibleToken(...)`:
- Convert `msg.Symbol` to lowercase
- Do not scale up `InitialSupply` and `MaxSupply`